src/NServiceBus.Transport.SQS/InputQueuePump.cs

```diff
@@ -66,14 +66,16 @@ public async Task Initialize(PushRuntimeSettings limitations, OnMessage onMessag
             this.onError = onError;
         }
 
-        public Task StartReceive(CancellationToken cancellationToken)
+        public Task StartReceive(CancellationToken cancellationToken = default)
         {
-            if (tokenSource != null)
+            if (messagePumpCancellationTokenSource != null)
```

Comment:
This is the only of our pumps that guard for multiple calls to start, should we add this to all the others?

Reply:
Same thing as for Stop: https://github.com/Particular/NServiceBus/blob/master/src/NServiceBus.Core/Hosting/HostCreator.cs#L75-L82